motoman_mh5_support/urdf/mh5.urdf

```diff
@@ -135,7 +140,12 @@
     <child link="link_u"/>
     <origin rpy="0 0 0" xyz="0 0 0.3100"/>
     <axis xyz="0 -1 0"/>
-    <limit effort="100" lower="-2.3736" upper="4.4506" velocity="2.96"/>
+    <!-- Old limits: lower="-2.3736" upper="4.4506" The new limits
+           below were measured from an MH5F.  They ensure that link_r
+           won't hit link_l even when joint_r is at 45 degrees.  The
+           old limits were converted directly from the MH5 data sheet,
+           which was clearly way off. -->
+    <limit effort="100" lower="-1.0122" upper="3.2218" velocity="2.96"/>
```

Comment:
I'll assume this change too was done previously and the srdf was just not pushed with this change?

Reply:
Yes